README.md

```diff
@@ -50,7 +50,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://danutibe07.github.io/math-magicians/)
+- [Not Availaible]()
```

Comment:
- [ ] It would be better to delete a not working live demo link or broken link in our README file, this link is not helping make our README professional, kindly remove it